src/CppUTest/TestRegistry.cpp

```diff
@@ -50,6 +50,7 @@ void TestRegistry::runAllTests(TestResult& result)
     result.testsStarted();
     for (UtestShell *test = tests_; test != NULL; test = test->getNext()) {
```

Comment:
Yeah, I don't like the setIgnoreTest in the interface, but for now I think it is ok.

As I mentioned, usually this kind of info is passed on creation, but in our case we cannot do that because the creation happens in the macro. So, for now its ok.